src/bp/nlu-core/training-worker-queue/index.ts

```diff
@@ -198,9 +213,11 @@ if (cluster.isMaster) {
     worker.process.kill('SIGKILL')
   }
 
-  registerMsgHandler('make_new_worker', async (msg: OutgoingMessage<'make_new_worker'>) =>
-    spawnNewTrainingWorker(msg.payload.config, msg.payload.requestId)
-  )
+  registerMsgHandler('make_new_worker', async (msg: OutgoingMessage<'make_new_worker'>) => {
+    debugLog('About to spawn new process.', msg.payload.requestId)
```

Comment:
Can you be a bit more clear in your debug message. Reading logs I'd love to know what that process will do, we might spawn other kind of processes in the future.

as simple as `About to spawn new training process` and `Done spawning new training process`

Reply:
And just wondering here, what's the reason why you added those 2 debug log. Is it in the case where we want to debug if the training process doesn't start and the following message doesn't appear `Training worker ${srcWorkerId} successfully started on process with pid ${processId}.` ? 

I guess there's no such things as too much debug

Reply:
Yep, this sometimes occur and I have absolutely no clue why it does + We can't reproduce the issue when we want. 

I'll add "training"